Makefile

```diff
@@ -339,6 +342,47 @@ shell-check:
 check: copyright-check lint shell-check
 
 ##### Tests #####
+collect-modified-tests:
```

Comment:
I would really like to see us writing less Makefile and bash scripts in general. I would encourage you to use Go for scripting wherever possible for readability, safety and platform compatibility.